app/models/podcast.rb

```diff
@@ -14,8 +14,18 @@ class Podcast < BaseModel
 
   acts_as_paranoid
 
+  before_validation :set_defaults
+
   scope :published, -> { where('published_at IS NOT NULL AND published_at <= now()') }
 
+  def set_defaults
+    self.enclosure_template ||= enclosure_template_default
+  end
+
+  def enclosure_template_default
+    "https://#{ENV['DOVETAIL_HOST']}/{slug}/{guid}/{original_filename}"
```

Comment:
yes, I am assuming all our feeds will go through dovetail by default. This could be revisited later.